spec/spec_helper.rb

```diff
@@ -28,11 +27,12 @@
 #   end
 # end
 
-Capybara.current_driver = :my_firefox_driver
+Capybara.default_driver = :my_firefox_driver
```

Comment:
This change was required to allow multiple specs to use the same browser session.